### PR TITLE
Fixed typo in "Customize the password reveal button"

### LINKED ISSUE #1612: Docs "Customize the password reveal button" Typo
In the opening paragraph there's a typo.

> The revealed password field helps the user verify if the password is **correctly.**

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5949a483-1a4d-0f77-5576-af0a335fc959
* Version Independent ID: 824cf57c-044d-9dac-085a-8bcfd6f2dc15
* Content: [Customize the password reveal button - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/web-platform/password-reveal)
* Content Source: [microsoft-edge/web-platform/password-reveal.md](https://github.com/MicrosoftDocs/edge-developer/blob/master/microsoft-edge/web-platform/password-reveal.md)
* Product: **microsoft-edge**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by SIkebe**
Fixes #1612

**Rendered page for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/password-reveal?branch=pr-en-us-1613

### CODE REVIEW

File: microsoft-edge/web-platform/password-reveal.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ The `password` input type in Microsoft Edge includes a **password reveal** contr
 
 By default, the **password reveal** button inserts into the Shadow DOM of all HTML `input` elements with the `type` set to `"password"`.  Starting with Microsoft Edge Version 87, users or [enterprises](/deployedge/microsoft-edge-policies#passwordrevealenabled) may disable this feature globally.  You, web designers and developers, should expect most Microsoft Edge users to have the default experience.
 
-
-<!-- ====================================================================== -->
+
+<!-- ====================================================================== -->
```
Comment (SIkebe) [inline microsoft-edge/web-platform/password-reveal.md:35]: Hmm...when I edited the page via browser, some line endings were fixed automatically. Should I revert these changes?

Comment (mikehoffms) [reply microsoft-edge/web-platform/password-reveal.md:35]: @mikehoffms - I'll scope out

Comment (mikehoffms) [reply microsoft-edge/web-platform/password-reveal.md:35]: Let's keep the changed line endings, that the tool made.

I wasn't able to reproduce that automatic change when I tried the "." browser-based VS Code.  I haven't determined what the "line ending" change amounts to.  Comment dividers are supposed to be using normal line endings already.  I didn't see such an auto-fix in a different PR when I used the browser-based VS Code.

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/web-platform/password-reveal.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ keywords: microsoft edge, compatibility, web platform, password reveal, eye icon
 ---
 # Customize the password reveal button
 
-The `password` input type in Microsoft Edge includes a **password reveal** control.  A user may choose the **password input** button to reveal the **password** field.  The revealed **password** field helps the user verify if the password is correctly.  After a user has entered text in the **password** field, a user may choose the **password reveal** button or select `Alt`+`F8` to toggle visibility of the input.
+The `password` input type in Microsoft Edge includes a **password reveal** control.  A user may choose the **password input** button to reveal the **password** field.  The revealed **password** field helps the user verify if the password is correct.  After a user has entered text in the **password** field, a user may choose the **password reveal** button or select `Alt`+`F8` to toggle visibility of the input.
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/password-reveal.md:13]:
```suggestion
The `password` input type in Microsoft Edge includes a **password reveal** control.  A user can click the **password reveal** button to show the characters in the password field, to make sure that the password is entered correctly.  After a user has entered text in the **password** field, a user can either click the **password reveal** button or select `Alt`+`F8` to toggle visibility of the input.

```
